actions/ui-check/package.json

```diff
@@ -8,6 +8,8 @@
 		"format:js": "wp-scripts format-js",
 		"test:e2e:interactive": "PUPPETEER_HEADLESS=false wp-scripts test-e2e --config tests/e2e/jest.config.js",
 		"test:e2e": "wp-scripts test-e2e --config tests/e2e/jest.config.js",
```

Comment:
I wonder if the e2e tests (also?) belong in the top-level package.json? That's where I initially assumed they were.